frontend/src/components/uam/components/users-table/users-table.tsx

```diff
@@ -27,7 +27,7 @@ const UsersTable: FC = () => {
         count: PaginationDefaultValue.DEFAULT_COUNT,
       }),
     );
-  }, [page]);
+  }, [page, users.total]);
```

Comment:
```suggestion
  }, [page, usersTotalCount]);
```
could you please the reducer stucrute? everything should be in flat way